app/views/catalog/librarian_view.html.erb

```diff
@@ -6,6 +6,11 @@
     </button>
   <% end %>
   <h3 class="modal-title"><%= t('blacklight.search.librarian_view.title') %></h3>
+  
+  <% if @document[:last_updated] %>
+    <p>Last updated in SearchWorks on <%= @document[:last_updated].to_time %></p>
```

Comment:
I'm guessing we'll want to format this value for display. I don't know that we've had to do that in searchworks yet, but we do in other projects, e.g.:

https://github.com/sul-dlss/mylibrary/blob/main/config/locales/en.yml#L33-L38

So I think it'd be safe to pull those over, and to use:
```suggestion
    <p>Last updated in SearchWorks on <%= l @document[:last_updated].to_time, format: :long %></p>
```